AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1915,6 +1916,8 @@ finanzen100.de##.newsletter-subscribe
 t-online.de##.teaser-cr-subscribe
 ||animedia.online/banner/youtube
 animedia.online#$#.film-desc { min-height: 86px!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/30385
```

Comment:
Please add a box like #### Whitelists ### above this rule, so we have a whitelist section there